db/schema.rb

```diff
@@ -22,6 +22,18 @@
     t.datetime "updated_at", precision: 6, null: false
   end
 
+  create_table "comments", force: :cascade do |t|
+    t.bigint "user_id"
+    t.string "commentable_type", null: false
+    t.bigint "commentable_id", null: false
+    t.integer "parent_id"
```

Comment:
If we use `commentable_type` and `commentable_id`, will we still need parent_id?

Reply:
I think we need it to know which is parent

Reply:
What's the different from reusing polymorphic columns like this?
```
  parent = Comment.create
  child = Comment.create(commentable: parent)
```